### PR TITLE
fix for issue#21:  "value" attribute not respected by dropdown input …

### PR DESCRIPTION
**Pull request by jjschultz**
I didn't mean to reformat the indent char for all of tests.py, but it happened.  I normall would revert and un-reformat, but I think the change brings it consistent with the rest of the code which uses tabs instead of spaces.

### CODE REVIEW

**Comment by jjschultz** [comment]
the reformatting of tests.py hides the actual change - I just added test_dropdown_value

File: spyre/server.py [hunk 1]
```diff
@@ -116,31 +116,36 @@ def use_custom_input_values(self, args):
...
-						if option['value'] in args[input['variable_name']]:
-							self.templateVars['inputs'][index]['options'][index2]['checked'] = True
-						else:
-							self.templateVars['inputs'][index]['options'][index2]['checked'] = False
-						index2+=1
+			if input['input_type'] in ['text','slider']:
+				self.templateVars['inputs'][index]['value'] = input_value
+			if input['input_type'] in ['radiobuttons', 'dropdown']:
+				for option in input['options']:
+					option['checked'] = (option['value'] == input_value)
```
Comment (adamhajari) [inline spyre/server.py]: elegant!

**Comment by adamhajari** [comment]
JJ this looks great! Happy to see that this solution works for both dropdowns and radiobuttons.

**Comment by jjschultz** [comment]
Sweet ­ thanks!  Very cool project ­ glad I could contribute

From:  Adam Hajari <notifications@github.com>
Reply-To:  adamhajari/spyre
<reply+00728ed9154b300a6fac539f97fe6b918d5b86125247ca0892cf0000000111a42f039
2a169ce056d7822@reply.github.com>
Date:  Thursday, June 25, 2015 at 5:05 PM
To:  adamhajari/spyre <spyre@noreply.github.com>
Cc:  Jerod 'JJ' Schultz <jj@jjschultz.com>
Subject:  Re: [spyre] fix for issue#21:  "value" attribute not respected by
dropdown input  (#27)


JJ this looks great! Happy to see that this solution works for both
dropdowns and radiobuttons.


Reply to this email directly or view it on GitHub
<https://github.com/adamhajari/spyre/pull/27#issuecomment-115398257> .

**Pull request merged**